hbase-shell/src/main/ruby/shell/commands/merge_region.rb

```diff
@@ -33,16 +33,18 @@ def help
 
 Examples:
 
-  hbase> merge_region 'FULL_REGIONNAME', 'FULL_REGIONNAME'
-  hbase> merge_region 'FULL_REGIONNAME', 'FULL_REGIONNAME', true
+  hbase> merge_region ['FULL_REGIONNAME', 'FULL_REGIONNAME']
+  hbase> merge_region ['FULL_REGIONNAME', 'FULL_REGIONNAME', 'FULL_REGIONNAME', ...]
+  hbase> merge_region ['FULL_REGIONNAME', 'FULL_REGIONNAME', 'FULL_REGIONNAME', ...], true
 
-  hbase> merge_region 'ENCODED_REGIONNAME', 'ENCODED_REGIONNAME'
-  hbase> merge_region 'ENCODED_REGIONNAME', 'ENCODED_REGIONNAME', true
+  hbase> merge_region ['ENCODED_REGIONNAME', 'ENCODED_REGIONNAME']
+  hbase> merge_region ['ENCODED_REGIONNAME', 'ENCODED_REGIONNAME', 'ENCODED_REGIONNAME', ...]
+  hbase> merge_region ['ENCODED_REGIONNAME', 'ENCODED_REGIONNAME', 'ENCODED_REGIONNAME', ...], true
```

Comment:
As my previous comment, if we keep support of the syntax for adding only two regions and multiple regions we should keep both examples.

Reply:
Done.